style.css

```diff
@@ -71,27 +58,35 @@ h2 {
 	text-align: center;
 	font-weight: bold;
 }
-/* .item {
-	flex-grow: 3; /* might need to set a width for child elements */
+
+h1 {
+	font-size: 3.1rem;
+	color: #ffe4e1;
```

Comment:
I think you've set the sane color in the nav selector.  So either keep this or keep the color in the nav selector but not both.